### PR TITLE
Cutadapt 1.15 version bump

### PR DESCRIPTION
**Pull request by marcelm**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/cutadapt/meta.yaml [hunk 1]
```diff
@@ -1,32 +1,31 @@
...
-  url: https://pypi.python.org/packages/16/e3/06b45eea35359833e7c6fac824b604f1551c2fc7ba0f2bd318d8dd883eb9/cutadapt-1.14.tar.gz
-  md5: 57403c3794aa940699df3032a5e9b208
+  fn: cutadapt-1.15.tar.gz
+  url: https://pypi.python.org/packages/5a/89/a041e0a108bef64b1347e9b7126d813e61f0ce028d3db830fb7b77aab1ab/cutadapt-1.15.tar.gz
+  md5: 50baff135228644e8c68e90e2bd4e953
 
 build:
   number: 0
-  script: $PYTHON setup.py install
+  script: pip install .
```
Comment (bgruening) [inline recipes/cutadapt/meta.yaml]: Hi @marcelm! Why did you changed this here? Please use `$PYTHON setup.py install`.

Comment (marcelm) [reply recipes/cutadapt/meta.yaml]: See http://www.python3statement.org/practicalities/ (fourth bullet point) and also https://youtu.be/2DkfPzWWC2Q?t=17m34s .

It probably doesn’t matter so much in a recipe, but I would like to stop using `setup.py` directly wherever I can.

Comment (bgruening) [reply recipes/cutadapt/meta.yaml]: I'm aware of this, but we try to align with the conda-forge guys atm and the current recommendation is to still use setup.py afaik.

**Comment by marcelm** [comment]
I made the requested change, is it ok now? Thanks!

**Comment by bgruening** [comment]
Perfect! Thanks.

**Pull request merged**